<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>928</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>741</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalcular">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalvarCSV">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar CSV</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>12</y>
      <width>181</width>
      <height>147</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
       <item>
        <widget class="QSpinBox" name="spinN"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Número de foguetes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEps">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Precisão desejada</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="r">
       <property name="text">
        <string>Gerar a automáticamente?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineAstart">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Valor inicial de a</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePasso">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Incremento de a</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1312</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFoguete">
    <property name="title">
     <string>Foguete</string>
    </property>
   </widget>
   <addaction name="menuFoguete"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
